pkg/services/ngalert/api/tooling/definitions/provisioning_alert_rules.go

```diff
@@ -87,7 +88,7 @@ type ProvisionedAlertRule struct {
 	// required: true
 	ExecErrState models.ExecutionErrorState `json:"execErrState"`
 	// required: true
-	For time.Duration `json:"for"`
+	For model.Duration `json:"for"`
```

Comment:
Hate to be that person, but since this now serializes as a string instead of a number in JSON, does this count as a compatibility break?

Luckily there is no TF provider calling this yet. If there was, we'd have some issues (version N-1 only supports grafana 9.0 - 9.1, version N only supports grafana 9.0+) and we'd need to sync releases.

Reply:
It never worked before, so I wouldn't tag this as a breaking change. If anybody would had used it -- they wouldn't be able to.